x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -200,10 +203,11 @@ export const InnerFieldItem = function InnerFieldItem(props: FieldItemProps) {
         container={document.querySelector<HTMLElement>('.application') || undefined}
         button={
           <DragDrop
+            noKeyboardSupportYet
+            draggable
             label={field.displayName}
             value={value}
-            data-test-subj={`lnsFieldListPanelField-${field.name}`}
-            draggable
+            dataTestSubj={`lnsFieldListPanelField-${field.name}`}
           >
             <FieldButton
```

Comment:
Can't leave a suggestion because it's too far away, sorry!

On line 222, can you change the `defaultMessage` to `Preview {fieldName}: {fieldType}`. 

Because we're using a standard `onClick` we don't need to specify how to activate a button.